docs/integrations/persisting-store-data.md

```diff
@@ -58,23 +75,16 @@ export const useBoundStore = create(
 )
 ```
 
-The given storage must match the following interface:
-
-```ts
-interface Storage {
-  getItem: (name: string) => string | null | Promise<string | null>
-  setItem: (name: string, value: string) => void | Promise<void>
-  removeItem: (name: string) => void | Promise<void>
-}
-```
-
 ### `serialize`
 
-> Schema: `(state: Object) => string | Promise<string>`
+> Type: `(state: Object) => string | Promise<string>`
```

Comment:
I replaced "Schema" with "Type", because a schema reminds me more
of an API schema in remote communication, like REST or GraphQL schema.